src/DataTable/tests/TableActions.test.jsx

```diff
@@ -245,4 +247,13 @@ describe('<TableActions />', () => {
       expect(icon.props().screenReaderText).toEqual(SMALL_SCREEN_DROPDOWN_BUTTON_TEXT);
     });
   });
+
+  describe('with function', () => {
+    it('passed correct number of selected rows', () => {
```

Comment:
We could probably include test case(s) for the added `PropTypes.arrayOf(PropTypes.func)` arguments.